packages/gafl-webapp-service/src/pages/summary/contact-summary/contact-summary.njk

```diff
@@ -44,10 +44,16 @@
 
             {{ csrf() }}
         </form>
-
-        <p class="govuk-body-m">
-            <a class="govuk-link" href="{{ data.uri.licenceSummary }}">{{ mssgs.change_licence_details }}</a>
-        </p>
+        
+        {% if data.isLicenceForYou %}
+            <p class="govuk-body-m">
+                <a class="govuk-link" href="{{ data.uri.licenceSummary }}">{{ mssgs.change_licence_details_you }}</a>
```

Comment:
Are you able to make this a one liner like:
`{% set title = mssgs.dob_title_you if data.isLicenceForYou else mssgs.dob_title_other %}
` ?